src/UNL/MediaHub/Media.php

```diff
@@ -255,17 +255,19 @@ function delete(Doctrine_Connection $conn = null)
      */
     public function getLocalFileName()
     {
-        $uploads_url = UNL_MediaHub_Controller::getURL() . 'uploads/';
-        if (strpos($this->url, $uploads_url) !== 0) {
+        $agnostic_file_url = preg_replace('/https?:\/\//', '//', $this->url, 1);
```

Comment:
Fixed in https://github.com/mfairchild365/UNL_MediaHub/commit/038d122ac1e8b50af6939e5067eb396e973272ca